### PR TITLE
feat(page): support ctrl-y shortcut for redo

### LINKED ISSUE #5120: Support Ctrl + Y as a shortcut key for Redo in Windows
### Describe the bug

The Ctrl+Y shortcut is not working for undo actions in desktop client

### To Reproduce

1.  Write some text and use the Ctrl+Z shortcut
2.  Then try using Ctrl+Y shortcut

### Platform
Windows 10

### Distribution version

Windows x64

### What browsers are you seeing the problem on if you're using web version?

_No response_

### Relevant log output

_No response_

### Anything else?

Also the zoom-in shortcut is not working on the desktop client Ctrl+ +

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

### PR DESCRIPTION
**Pull request by lzlme**
resolve #5120

### CODE REVIEW

File: tests/utils/actions/keyboard.ts [hunk 1]
```diff
@@ -97,7 +97,11 @@ export async function formatType(page: Page) {
   });
 }
 
-export async function redoByKeyboard(page: Page) {
+export async function redoByKeyboard(page: Page, onlyWin = false) {
+  if (onlyWin && IS_WINDOWS) {
```
Comment (doodlewind) [inline tests/utils/actions/keyboard.ts:101]: This is not a valid test, `IS_WINDOWS` should be always false on CI. Given the situation that cross-OS testing suite is not set up, not testing this PR is just fine.

File: packages/blocks/src/page-block/keyboard/keyboard-manager.ts [hunk 2]
```diff
@@ -18,6 +18,12 @@ export class PageKeyboardManager {
             this._page.redo();
           }
         },
+        'Control-y': ctx => {
```
Comment (doodlewind) [inline packages/blocks/src/page-block/keyboard/keyboard-manager.ts:21]: Should make this to be triggered windows only.

File: packages/blocks/src/page-block/keyboard/keyboard-manager.ts [hunk 3]
```diff
@@ -18,6 +19,14 @@ export class PageKeyboardManager {
...
+          if (IS_WINDOWS) {
```
Comment (doodlewind) [inline packages/blocks/src/page-block/keyboard/keyboard-manager.ts:23]: We prefer the `if (!IS_WINDOWS) return` guard pattern, this lowers the code nesting and makes condition more straightforward to understand.

Comment (lzlme) [reply packages/blocks/src/page-block/keyboard/keyboard-manager.ts]: I just thought about whether to write it like this haha, I will change it now

**Review by doodlewind — APPROVED** [review]

**Pull request merged**